### PR TITLE
Cmake: Made it possible to specify external modules and projects with backslash

### PR DESCRIPTION
**Pull request by ResearchDaniel**
inviwomodulepaths.h would be incorrect if external module paths were specified with \path\to\folder

### CODE REVIEW

File: cmake/globalconfig.cmake [hunk 1]
```diff
@@ -99,11 +99,11 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_LIST_DIR}/modules")
 
 #--------------------------------------------------------------------
 # Add parameter for paths to external modules
-set(IVW_EXTERNAL_MODULES "" CACHE STRING "Paths to directory containing external modules")
+set(IVW_EXTERNAL_MODULES "" CACHE STRING "Semicolon (;) separated paths to directories containing external modules")
 
 #--------------------------------------------------------------------
 # Add parameter for paths to external projects
-set(IVW_EXTERNAL_PROJECTS "" CACHE STRING "Paths to directory containing external projects")
+set(IVW_EXTERNAL_PROJECTS "" CACHE STRING "Semicolon (;) separated paths to directories containing cmake config file (.cmake/CMakeLists.txt file)")
```
Comment (petersteneteg) [inline cmake/globalconfig.cmake]: What do you mean by .cmake/?

File: cmake/globalmacros.cmake [hunk 2]
```diff
@@ -91,6 +91,7 @@ endfunction()
 function(ivw_generate_module_paths_header)
     set(dirs "")
     foreach(dir ${IVW_MODULE_DIR} ${IVW_EXTERNAL_MODULES})
+		file(TO_CMAKE_PATH "${dir}" dir)
```
Comment (petersteneteg) [inline cmake/globalmacros.cmake]: Indentation seems off...

File: cmake/globalconfig.cmake (hunk 1 above)
Comment (petersteneteg) [reply cmake/globalconfig.cmake]: Should probably not call CMakeLists.txt files Cmake config file as that might be confused with FooConfig.cmake files...

**Review by petersteneteg — APPROVED** [review]

File: cmake/globalconfig.cmake (hunk 1 above)
Comment (ResearchDaniel) [reply cmake/globalconfig.cmake]: I'll adjust and say that add_subdirectory will be called

File: cmake/globalmacros.cmake (hunk 2 above)
Comment (ResearchDaniel) [reply cmake/globalmacros.cmake]: Fixed

**Pull request merged**